dc/s2n-quic-dc/events/map.rs

```diff
@@ -19,6 +19,9 @@ struct PathSecretMapUninitialized {
     /// The number of entries in the map
     #[measure("entries")]
     entries: usize,
+
+    #[measure("lifetime")]
```

Comment:
Just to check, we auto-unit durations?

Reply:
oh we default to `None`... lemme fix that. good catch!